swgohhelp/client.go

```diff
@@ -158,31 +161,32 @@ func (c *Client) Players(allyCodes ...string) (players []Player, err error) {
 		return nil, err
 	}
 
-	// Enrich result with related data from Crinolo's stat API
-	url := "https://crinolo-swgoh.glitch.me/statCalc/api/characters/?flags=withModCalc,gameStyle"
-	for i := range players {
-		player := players[i]
-		b, err := json.Marshal(player.Roster)
-		if err != nil {
-			return nil, err
-		}
-		if c.debug {
-			writeLogFile(b, "req", "POST", "_crinoloapi")
-		}
-		resp, err := c.hc.Post(url, "application/json", bytes.NewBuffer(b))
-		if err != nil {
-			return nil, err
-		}
-		b, err = ioutil.ReadAll(resp.Body)
-		if err != nil {
-			return nil, err
-		}
-		if c.debug {
-			writeLogFile(b, "resp", "POST", "_crinoloapi")
-		}
-		err = json.Unmarshal(b, &player.Roster)
-		if err != nil {
-			return nil, err
+	if (*useExternalStats) {
+		// Enrich result with related data from Crinolo's stat API
+		url := "https://swgoh-stat-calc.glitch.me/api/?flags=withModCalc,gameStyle,calcGP"
```

Comment:
This is the main change (besides adding the flag to allow disabling of the stats calc API).  It uses the new version of the API which has improved support for calculating GP including relics.

Reply:
LGTM